acme-stack/src/main/packaging/sample/content/multiple-forms.html

```diff
@@ -45,9 +45,9 @@ <h2>
   </div>
   <div class="row">
     <div class="col-md-12">
-      <script data-knotx-knots="form-competition,services,handlebars"
-              data-knotx-service-labelsrepository="labelsRepository"
-              data-knotx-params-labelsrepository='{"path":"/service/labelsRepository.json"}'
+      <script data-knotx-knots="form-competition,databridge,handlebars"
+              data-knotx-databridge-name-labelsrepository="labelsRepository"
```

Comment:
I don't like `name` here - this doesn't represent either data adapter or definition.
Let's discuss it in a separate ticket.